ironfish/src/rpc/routes/wallet/burnAsset.ts

```diff
@@ -60,10 +62,15 @@ router.register<typeof BurnAssetRequestSchema, BurnAssetResponse>(
       throw new ValidationError('Invalid burn amount')
     }
 
+    const assetBuffer = Buffer.from(request.data.assetId, 'hex')
+
+    const assetResponse = await node.chain.getAssetById(assetBuffer)
```

Comment:
@ygao76 do we plan to change the data included in `BurnDescription` as part of offline signing?

If requests and/or responses include the asset name then we may not need to look up the asset here.